go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/privacybydesign/gabi
 
-go 1.18
+go 1.21
+
+toolchain go1.21.1
```

Comment:
The `toolchain` tag is not really relevant here, because `gabi` is only used as library for other Go projects and in those scenarios the `toolchain` tag is ignored. If I'm right you did not actually change something that requires Go 1.21, so we could also consider sticking to the language features of Go 1.18 for now.